arches_references/datatypes/datatypes.py

```diff
@@ -76,8 +79,28 @@ def validate(
         return errors
 
     def transform_value_for_tile(self, value, **kwargs):
+        controlled_list = kwargs.get("controlledList")
+        nodeid = kwargs.get("nodeid")
+        config = {"controlledList": controlled_list, "nodeid": nodeid}
+        if type(value) == str and config:
+            value = [self.lookup_listitem_from_label(value, config).build_tile_value()]
         return value
 
+    def lookup_listitem_from_label(self, value, config):
+        if "controlledList" in config:
+            list_id = config["controlledList"]
+        elif "nodeid" in config:
+            nodeid = config["nodeid"]
```

Comment:
Theoretically, the two values should be the same, and since a reference datatype node can only use 1 controlled list, I think we could just guard against cases when the listid is not the same as the listid assigned to the node.